sunpy/physics/tests/test_differential_rotation.py

```diff
@@ -43,6 +45,22 @@
 # was largely based on that in SSWIDL.
 
 
+testpath = sunpy.data.test.rootdir
+
+
+@pytest.fixture
+def aia171_test_map():
+    return sunpy.map.Map(os.path.join(testpath, 'aia_171_level1.fits'))
+
+
+@pytest.fixture
+def aia171_test_map_with_mask(aia171_test_map):
+    shape = aia171_test_map.data.shape
+    mask = np.zeros_like(aia171_test_map.data, dtype=bool)
+    mask[0:shape[0]//2, 0:shape[1]//2] = True
+    return sunpy.map.Map(np.ma.array(aia171_test_map.data, mask=mask), aia171_test_map.meta)
+
+
```

Comment:
Ah no, are the starting point from @wafels to create tests on them... but he never did...